docs/developers/meta/sc-meta-cli.md

```diff
@@ -209,6 +209,30 @@ Paramameter:
     -  The framework version on which the contracts should be created.
     - _default_: The latest released version.
 
+[comment]: # (mx-context-auto)
+
+### Calling `test`
+
+This command runs `cargo test`, which builds the project and runs the tests.
+
+Parameters:
+- `--path`  
+    - Target directory where to generate contract integration tests.
+    - _default_: current directory.
+- `--go`            
+    - This arg runs rust and go tests.
```

Comment:
Use this argument to only run the `mx-scenario-go` tool, directly. It is equivalent to running `mx-scenario-go run ...`

Maybe post a link on how to install mx-scenario-go.